application/src/main/resources/thingsboard.yml

```diff
@@ -768,7 +768,28 @@ spring:
       leakDetectionThreshold: "${SPRING_DATASOURCE_HIKARI_LEAK_DETECTION_THRESHOLD:0}"
       # This property increases the number of connections in the pool as demand increases. At the same time, the property ensures that the pool doesn't grow to the point of exhausting a system's resources, which ultimately affects an application's performance and availability
       maximumPoolSize: "${SPRING_DATASOURCE_MAXIMUM_POOL_SIZE:16}"
-      registerMbeans: "${SPRING_DATASOURCE_HIKARI_REGISTER_MBEANS:false}" # true - enable MBean to diagnose pools state via JMX
+      # Enable MBean to diagnose pools state via JMX
+      registerMbeans: "${SPRING_DATASOURCE_HIKARI_REGISTER_MBEANS:false}"
+    dedicated:
```

Comment:
We need to decide on a better naming here. Assuming we will have separate data sources for partitioned attributes & latest ts, we should choose naming now. 

I assume an array:

```
datasources:
  - name: main
     ...
  - name: audit_logs_and_events
     ...
  - name: telemetry
    partition: 1
    ...
```

Reply:
We need to make sure we can configure everything using env variables (docker + k8s)

Reply:
events: